src/sparql/exact_syn_from_label.ru

```diff
@@ -0,0 +1,11 @@
+# Create an exact synonym copy for all labels
```

Comment:
## Create an exact synonym copy for all labels

Reply:
- [ ] @matentzn and/or @twhetzel Is this really the best way to do this?

My builds are taking a very long time and not yet completing. It's seemed to freeze twice. I think this is because the laptop accidentally went to sleep both times.

But it made me think of something: Maybe this change has unintended consequences.

It would be very easy for me to handle this `label` --> `exactSynonym` thing in the subclass sync Python script, without needing to modify these components.
I am wondering if adding all these extra synonyms is resulting in any of: (i) potentially causing lexmatch to take longer, (ii) might cause other lexmatch issues, (iii) might cause other unforeseen issues.

When the build (#601) is ready, let's look at the outputs and see about (ii) and (iii).
However if the build is really taking longer because of this, I recommend we do this in Python instead.

Reply:
Given this comment and the Slack comment, I will also try to run a build from this feature branch.

Reply:
Scratch my comment earlier about "changing the SPARQL to a Python implementation", as [Nico said](https://monarchinitiative.slack.com/archives/C03AW548J8M/p1720094791335579?thread_ts=1719948621.736089&cid=C03AW548J8M):
> ...the general pattern (taking the label and asserting as a synonym) is used for lexical matching...

Basically if this not only works for synonym sync, but not only doesn't hurt, but actually helps with lexmatch, we should keep this implementation.